microceph/ceph/services.go

```diff
@@ -5,12 +5,105 @@ import (
 	"database/sql"
 	"fmt"
 
+	"github.com/Rican7/retry"
+	"github.com/Rican7/retry/backoff"
+	"github.com/Rican7/retry/strategy"
 	"github.com/canonical/microcluster/state"
 
 	"github.com/canonical/microceph/microceph/api/types"
 	"github.com/canonical/microceph/microceph/database"
+	"github.com/tidwall/gjson"
 )
 
+type Set map[string]int
```

Comment:
Nit: this is often done like the below because we don't actually need an `int` here, we only need to know that a key is present, and `struct{}` is smaller

```
type Set map[string]struct{}
...
var exists = struct{}{}
mySet["foo"] = exists
```